src/actions/app.js

```diff
@@ -0,0 +1,48 @@
+import actionTypes from '../constants/actionTypes';
```

Comment:
Maybe name it as `participants.js` so it's not named `app.js`?

Reply:
got it!